it/uk/gov/hmrc/transitmovementsrouter/utils/WiremockSuite.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This class is not required and should be removed.

Reply:
Better put, this is duplicated. It's in `it/base` -- we should be using that version and not introducing new ones.

Reply:
Done